apps/dialtone-documentation/docs/.vuepress/baseComponents/AllTokens.vue

```diff
@@ -14,50 +52,243 @@
       :options="THEMES"
       @change="setTheme"
     />
-  </dt-stack>
-  <div v-for="category in categories" :key="category">
-    <slot :name="category">
-      <!-- slot for anchor header -->
-    </slot>
-    <token-table :category="category" :format="format" :theme="theme" />
+  </div>
+  <div v-for="category in Object.keys(filteredTokens)" :key="category">
+    <h2
+      :id="category"
+      class="d-docsite--header-2 d-tt-capitalize"
+      tabindex="-1"
+    >
+      <a
+        :href="`#${category}`"
+        class="header-anchor d-link d-docsite--link d-link"
+      >#</a>
+      {{ category }}
+    </h2>
+    <p class="dialtone-intro">
+      {{ categoryIntroAndDesc[category]?.intro }}
+    </p>
+    <div v-html="categoryIntroAndDesc[category]?.description" />
+    <token-table :category="category" :tokens="filteredTokens[category]" />
+  </div>
+  <div
+    v-if="!hasSearchResults"
```

Comment:
Here also we are using `hasSearchResults` just once and with a `!`.
I will suggest to update the name to sth like:

`noSearchResults`
`const noSearchResults = computed(() => Object.keys(filteredTokens.value).length === 0);`